eval_jumpstart/README.md

```diff
@@ -20,122 +22,85 @@ Create a YAML configuration file to specify the evaluation settings.
+
+llms:
+  - name: sncloud-llama3.1-405
+    model_type: "sncloud"
+    model_name: "Meta-Llama-3.1-405B-Instruct"
+    max_tokens: 1024
+    temperature: 0.0
+  - name: sambastudio-llama2-70
+    model_type: "sambastudio"
+    model_name: "llama-2-70b-chat-hf"
+    max_tokens: 1024
+    temperature: 0.0
+  - name: sncloud-llama3.1-70
+    model_type: "sncloud"
+    model_name: "Meta-Llama-3.1-70B-Instruct"
+    max_tokens: 1024
+    temperature: 0.0
+  - name: sncloud-llama3.2-3
+    model_type: "sncloud"
+    model_name: "Meta-Llama-3.2-3B-Instruct"
+    max_tokens: 1024
+    temperature: 0.0
+
+eval_llm:
+  model_type: "sncloud"
+  model_name: "Meta-Llama-3.1-405B-Instruct"
+  max_tokens: 1024
+  temperature: 0.0
 ```
 
 ## Evaluation Use Cases
 
 ```python
-## Use Case 1: Evaluation without Generation
-
-from rag_eval import RAGEvaluator, RAGEvalConfig
-from langchain_community.llms.sambanova import SambaStudio
-from langchain_community.embeddings import HuggingFaceInstructEmbeddings
-import pandas as pd
-
-config_path = "path/to/config.yaml"
-config = RAGEvalConfig(config_path)
-eval_llms = [
-    (llm_name, SambaStudio(**llm_config))
-    for conf in config.eval_llm_configs 
-    for llm_name, llm_config in [config.get_llm_config(conf)]
-]
-eval_embeddings = HuggingFaceInstructEmbeddings(model_name=config.embedding_model_name)
-evaluator = RAGEvaluator(eval_llms=eval_llms, eval_embeddings=eval_embeddings, config_yaml_path=config_path)
-eval_df = pd.read_csv("path/to/eval.csv")
-results_no_gen = evaluator.evaluate(eval_df)
-print(results_no_gen)
-```
+from dotenv import load_dotenv
+import sys
+import os
 
-## Use Case 3: Evaluation with Generation
+current_dir = os.getcwd()
+kit_dir = os.path.abspath(os.path.join(current_dir, '..'))
+repo_dir = os.path.abspath(os.path.join(kit_dir, '..'))
+sys.path.append(kit_dir)
+sys.path.append(repo_dir)
 
-```python
-from rag_eval import RAGEvaluator, RAGEvalConfig, load_pipeline
-from langchain_community.llms.sambanova import SambaStudio
-
-config_path = "path/to/config.yaml"
-config = RAGEvalConfig(config_path)
-eval_llms = [
-    (llm_name, SambaStudio(**llm_config))
-    for conf in config.eval_llm_configs 
-    for llm_name, llm_config in [config.get_llm_config(conf)]
-]
-eval_embeddings = HuggingFaceInstructEmbeddings(model_name=config.embedding_model_name)
-evaluator = RAGEvaluator(eval_llms=eval_llms, eval_embeddings=eval_embeddings, config_yaml_path=config_path)
-eval_df = pd.read_csv("path/to/eval.csv")
-pipelines = [
-    load_pipeline((llm_name, SambaStudio(**llm_config)), config)
-    for llm_name, llm_config in [
-        config.get_llm_config(conf) for conf in config.llm_configs
-    ]
-]
-results_with_gen = evaluator.evaluate(eval_df, pipelines)
-print(results_with_gen)
-```
+load_dotenv(os.path.join(repo_dir, '.env'), override=True)
 
-More use cases are available in the [notebooks](./notebooks)
+from utils.eval.evaluator import BaseWeaveEvaluator
+import weave
```

Comment:
shouldn't we use this patch instead of this line?
```
from utils.visual.env_utils import get_wandb_key

wandb_api_key = get_wandb_key()
if wandb_api_key:
    import weave
else:
    print('WANDB_API_KEY is not set. Weave initialization skipped.')
```

Reply:
Does this need to be resolved @snova-luiss or has it been already? Cheers!